pkg/projects/Microsoft.NETCore.DotNetHost/win/Microsoft.NETCore.DotNetHostResolver.pkgproj

```diff
@@ -13,7 +13,7 @@
   
   <ItemGroup>
     <Dependency Include="Microsoft.NETCore.DotNetHost">
-      <Version>$(HostFullVersion)</Version>
```

Comment:
This is just renaming. No physical change -- it is confusing to have the name as `FullVersion` and have a targeted fix under the name.